tests/boardfarm_plugins/boardfarm_prplmesh/tests/ap_config_renew.py

```diff
@@ -35,7 +32,7 @@ def runTest(self):
                                             tlv(0x0F, 0x0001, "{0x00}"),
                                             tlv(0x10, 0x0001, "{0x00}"))
 
-        time.sleep(30)
+        time.sleep(5)
```

Comment:
(nitpick) I think this should be in fixed in another PR but opening a PR just for this is too much.